test/integration/features/story_health.feature

```diff
@@ -54,6 +54,15 @@ Feature:
         And executing "oc expose svc/httpd-ex" succeeds
         And with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" has status code "200"
 
+    @darwin @linux
+    Scenario: Stop and start CRC, then check app still runs
+        Given with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" has status code "200"
+        When executing "crc stop" succeeds
+        Then with up to "4" retries with wait period of "2m" command "crc status" output should contain "Stopped"
+        When starting CRC with default bundle and default hypervisor succeeds
+        Then with up to "4" retries with wait period of "2m" command "crc status" output should match ".*Running \(v\d+\.\d+\.\d*\).*"
+        And with up to "2" retries with wait period of "60s" http response from "http://httpd-ex-testproj.apps-crc.testing" has status code "200"s
+
```

Comment:
why the 's' at the end of the line?

Reply:
@cfergeau That's from sleeping on the keyboard. I wonder why the test didn't fail on this (will check). Thanks for spotting it!